httpstan/models.py

```diff
@@ -8,9 +8,11 @@
 import hashlib
 import importlib
 import os
+import shutil as _shutil
```

Comment:
Would you mind telling me why these are imported with underscore-prefixed aliases?

Reply:
Just a copy-paste error